app/assets/stylesheets/earthworks.css

```diff
@@ -2,6 +2,16 @@
   --stanford-50-black: #979694;
 }
 
+
+.btn-outline-secondary {
```

Comment:
I suspect it's going to get confusing that we're using blue as the secondary color, since the component library defines blue as the primary color.   Is it possible to change the classes on the buttons?

Reply:
Not without changing backlight. Specifically it is these buttons: https://github.com/projectblacklight/blacklight/blob/a8b5a3d3a2e8243c8a875d40528eae5c106d83bd/app/helpers/blacklight/component_helper_behavior.rb#L26C1-L30C8  so maybe there is a way to override this class? I need to check to see what actions can do.

Reply:
I think that's why I added https://github.com/projectblacklight/blacklight/pull/3267

Reply:
makes sense. Will leave as is for now and wait until this gets merged.